source/contribute/maintainer.rst

```diff
@@ -59,16 +67,167 @@ Optionally, add any additional contributor/maintainer forks, e.g.:
 
     $ git remote add romanb git://github.com/romanb/doctrine2.git
 
+You can also fetch all open pull requests via `git fetch` if you
+edit your ``.git/config`` as following:
+
+... code-block:: console
+
+    [remote "doctrine"]
+        url = git@github.com:doctrine/doctrine2.git
+        fetch = +refs/heads/*:refs/remotes/doctrine/*
+        # add this:
+        fetch = +refs/pull/*/head:refs/remotes/doctrine/pr/*
+
+Distinguishing features, bugs and critical fixes
+------------------------------------------------
+
+The primary role of a maintainer, besides being also a contributor,
+is to sort incoming proposals by their category:
```

Comment:
Should we link to the specific policy pages for security fixes and deprecations in the respective categories?

Reply:
Yes. Can you suggest the links in a comment, please?

Reply:
I've added two Github suggestion comments with the links.